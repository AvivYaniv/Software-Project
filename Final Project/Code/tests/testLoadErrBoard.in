solve tests/b2_3_err.txt
print_board
mark_errors 0
solve tests/b2_3_err.txt
print_board
mark_errors 1
solve tests/b2_3_err.txt
print_board
edit tests/b2_3_err.txt
print_board
mark_errors 0