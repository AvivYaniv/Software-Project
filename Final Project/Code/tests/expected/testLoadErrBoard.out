Sudoku
------
Enter your command:
---------------------------
|  1.  2.    |          1*|
|  4.        |      3     |
---------------------------
|            |            |
|            |            |
---------------------------
|      1     |            |
|            |            |
---------------------------
Enter your command:
---------------------------
|  1.  2.    |          1*|
|  4.        |      3     |
---------------------------
|            |            |
|            |            |
---------------------------
|      1     |            |
|            |            |
---------------------------
Enter your command:
Enter your command:
---------------------------
|  1.  2.    |          1 |
|  4.        |      3     |
---------------------------
|            |            |
|            |            |
---------------------------
|      1     |            |
|            |            |
---------------------------
Enter your command:
---------------------------
|  1.  2.    |          1 |
|  4.        |      3     |
---------------------------
|            |            |
|            |            |
---------------------------
|      1     |            |
|            |            |
---------------------------
Enter your command:
Enter your command:
---------------------------
|  1.  2.    |          1*|
|  4.        |      3     |
---------------------------
|            |            |
|            |            |
---------------------------
|      1     |            |
|            |            |
---------------------------
Enter your command:
---------------------------
|  1.  2.    |          1*|
|  4.        |      3     |
---------------------------
|            |            |
|            |            |
---------------------------
|      1     |            |
|            |            |
---------------------------
Enter your command:
---------------------------
|  1*  2     |          1*|
|  4         |      3     |
---------------------------
|            |            |
|            |            |
---------------------------
|      1     |            |
|            |            |
---------------------------
Enter your command:
---------------------------
|  1*  2     |          1*|
|  4         |      3     |
---------------------------
|            |            |
|            |            |
---------------------------
|      1     |            |
|            |            |
---------------------------
Enter your command:
ERROR: invalid command
Enter your command:
Exiting...
